### PR TITLE
[7.26.x] JBPM-8758: [Stunner] BPMN process can't be saved if opened in parallel with DMN

### PR DESCRIPTION
**Pull request by karreiro**
See https://issues.jboss.org/browse/JBPM-8758

### CODE REVIEW

**Comment by hasys** [comment]
Jenkins execute full downstream build

**Comment by karreiro** [comment]
Jenkins execute full downstream build.

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by karreiro** [comment]
Jenkins execute full downstream build.

**Comment by manstis** [comment]
Jenkins execute full downstream build.

File: kie-wb-common-dmn/kie-wb-common-dmn-project-client/src/main/java/org/kie/workbench/common/dmn/project/client/editor/DMNDiagramEditor.java [hunk 1]
```diff
@@ -240,7 +240,9 @@ public void hideDocks() {
     }
 
     public void onMultiPageEditorSelectedPageEvent(final @Observes MultiPageEditorSelectedPageEvent event) {
-        searchBarComponent.disableSearch();
+        if (isSameSession()) {
+            searchBarComponent.disableSearch();
+        }
```
Comment (jomarko) [inline kie-wb-common-dmn/kie-wb-common-dmn-project-client/src/main/java/org/kie/workbench/common/dmn/project/client/editor/DMNDiagramEditor.java]: @karreiro @manstis hello, my question, shouldn't we update in the same way also `onRefreshFormPropertiesEvent` ?

Comment (manstis) [reply kie-wb-common-dmn/kie-wb-common-dmn-project-client/src/main/java/org/kie/workbench/common/dmn/project/client/editor/DMNDiagramEditor.java]: I'll have to defer that to @karreiro but I agree it looks plausible that both should be handled the same.

Comment (jomarko) [reply kie-wb-common-dmn/kie-wb-common-dmn-project-client/src/main/java/org/kie/workbench/common/dmn/project/client/editor/DMNDiagramEditor.java]: I checked manually the behavior, not sure if it related to my comment, but for example when you have a DMN diagram opened, you search for some node, then start manually clicking on nodes randomly, the search component is still shown, is it expected behavior?

Comment (jomarko) [reply kie-wb-common-dmn/kie-wb-common-dmn-project-client/src/main/java/org/kie/workbench/common/dmn/project/client/editor/DMNDiagramEditor.java]: @karreiro thank you for investigation

**Pull request merged**